### ISSUE TITLE
Update to Swift 4

### ISSUE THREAD

**Issue by welljsjs** [httpswift/swifter#276]
Hey, 

the current Swifter version is not optimized for Swift 4; when building my Xcode project, I get 19 warnings and 3 errors related to the Swifter framework, f.ex. 

`'characters' is deprecated: Please use String or Substring directly.`

The three errors: 
(All within "Scopes.swift"): 
```
scope(): Missing argument parameter #1 in call
c(): Missing argument parameter #1 in call
output = output + mergedAttributes.reduce("") {...
     Contextual closure type '(_, _) -> _' expects 2 arguments, but 1 was used in closure body.

```

Is there any fix or something I could do in order to make Swifter work?

Thanks a lot for help in advance!

EDIT: 

Just saw that there are already many requests regarding Swift 4 : #266 

Is there anyone who has power to merge?

**Comment by AlkiviadisPapadakis-je** [comment]
Any update on this?

**Comment by welljsjs** [comment]
Seems to be fixed : #266
